rules/windows/process_creation/proc_creation_win_expand_cabinet_files.yml

```diff
@@ -27,6 +27,7 @@ detection:
             - '\AppData\Local\Temp'
             - '\AppData\Roaming\Temp'
             - ':\Windows\Temp'
+            - '\\\\*$'
```

Comment:
hi,
what do you think about this way
```yaml
CommandLine|contains|all:
    - ' \\'
    - '$\'
```

Reply:
This will also use wildcards and is not of big benefits.